cli/connect/console.go

```diff
@@ -128,6 +129,11 @@ func (console *Console) Run() error {
 
 	console.prompt.Run()
 
+	// Sets the terminal modes to “sane” values to workaround
+	// bug https://github.com/c-bata/go-prompt/issues/228
+	reinitialization := exec.Command("reset")
```

Comment:
Nit: reset is not good way to fix this. It will clear previous stdin|stdout. 
The user may not expect this behavior.
It's good practice to restore the terminal without clearing the screen